### ISSUE TITLE
[Suggestion] ucname

### ISSUE THREAD

**Issue by danitag78** [danrevah/ngx-pipes#135]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
```

* **Ngx-pipes version:** any version

**Current behavior**

**Expected behavior**

ucwords or ucfirst pipes are not correct to print a name with - or ' chars.
I suggest to introduce an ucname pipe like [this](http://php.net/manual/en/function.ucwords.php#96179)

**Comment by JinnaBalu** [comment]
i think we should not do that for `, but can do i for -

**Comment by danitag78** [comment]
What about Shaquille O'Neal?
If I use ucwords, print Shaquille O'neal and it's not correct.

**Comment by danrevah** [comment]
Thanks!

Released a fix in `v2.4.3`.
